tryvital/db.py

```diff
@@ -0,0 +1,27 @@
+from contextlib import contextmanager
+import sqlite3
+from fastapi import Depends
+from sqlalchemy import text
+from sqlalchemy.ext.asyncio import AsyncEngine, create_async_engine, AsyncSession, async_sessionmaker
+
+DATABASE_URL = "sqlite+aiosqlite:///app.db"
+async_engine = create_async_engine(DATABASE_URL, echo=True)
+
+
+def get_engine() -> AsyncEngine:
+    return async_engine
+
+
+def get_session(db: AsyncEngine = Depends(get_engine)) -> AsyncSession:
+    session_maker: async_sessionmaker[AsyncSession] = async_sessionmaker(async_engine, expire_on_commit=False)
+    return session_maker()
```

Comment:
We would provide a simple `query()` function that takes text SQL statement and return the rows. Not everyone is familiar with SQLAlchemy, let alone the ORM feature.